### DISCUSSION TITLE
Cannot sign in

### DISCUSSION THREAD

**Discussion by peteringemann** [Tasshack/dreame-vacuum#402] [Q&A] [answered]
Hi there,
So this might be on me.
I bought a L10S Ultra (which is supported), installed the Dreamehome app. Then next i installed this application to my HomeAssistant and then, when prompted, I entered my email as username and password and chose the right server. 
But I get the 'Could not login to Xiaomi Miio Cloud, check the credentials.'
The credentials are correct. I even reset the password. 
Am I missing something? I feel like I need to create some sort of a cloud account at Xaiomi or something.

**Comment by Tasshack** [comment, marked as answer]
You are trying to login to the xiaomi cloud using your dreamecloud credentials.
You should install the Beta that has support for Dreamehome account.

**Reply by peteringemann** [reply]
Stupid question: how do I remove my previous installation (so I can download the beta - I take that the beta doesn't run very well along the master-tier release)

**Reply by Tasshack** [reply]
HACS has recently removed the version selector from the installation screen and they did not updated the documentation yet, so I don't know how to actually upgrade already installed integration to the Beta version on the lastest HACS.
Beta does not mean it does not work well or have some bugs, it means it does not tested by the enough people yet.

**Reply by peteringemann** [reply]
So I guess I'll just have to wait until current beta gets to rtm.

**Reply by Tasshack** [reply]
Then you will be waiting for a long time because download numbers are too low because of people thinks like you.

By the way I found the new way to upgrade an integration to a specific version.
https://experimental.hacs.xyz/docs/entities/update_entities/#install-service
Just type in the version as `v2.0.0b11` and it will be good to go.
